### PR TITLE
Fix deprecated note for service bus attributes

### PR DESCRIPTION
**Pull request by joaopgrassi**
## Changes

While working on the messaging migration guide, I noticed the deprecated attribute `messaging.servicebus.destination.subscription_name` mentions the wrong attribute that replaces it. 

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

**Comment by joaopgrassi** [comment]
CC @lmolkova

**Review by lmolkova — APPROVED** [review]
Thank you!

File: model/messaging/deprecated/registry-deprecated.yaml [hunk 1]
```diff
@@ -49,10 +49,10 @@ groups:
       - id: messaging.servicebus.destination.subscription_name
         type: string
         brief: >
-          Deprecated, use `messaging.servicebus.destination.subscription_name` instead.
+          Deprecated, use `messaging.destination.subscription.name` instead.
```
Comment (jsuereth) [inline model/messaging/deprecated/registry-deprecated.yaml:52]: Should we stop modifying both `brief` and `deprecated` and instead update markdown gen to fill in deprecated language for brief?

**Review by jsuereth — APPROVED** [review]

File: model/messaging/deprecated/registry-deprecated.yaml (hunk 1 above)
Comment (joaopgrassi) [reply model/messaging/deprecated/registry-deprecated.yaml:52]: Yes that would be great. A step towards make it standarized!

**Pull request merged**